CircuitPython_Pyloton/code.py

```diff
@@ -0,0 +1,59 @@
+from time import time
+import adafruit_ble
+import board
+import pyloton
+
+# PyLint can't find BLERadio for some reason so special case it here.
+ble = adafruit_ble.BLERadio()    # pylint: disable=no-member
+
+display = board.DISPLAY
+
+
+heart = True
+speed = True
+cad = True
+ams = True
+debug = False
+
+# 84.229 is wheel circumference (700x23 in my case)
+pyloton = pyloton.Pyloton(ble, display, 84.229, heart, speed, cad, ams, debug)
+
+pyloton.show_splash()
+
+ams = pyloton.ams_connect()
+
+
+start = time()
+hr_connection = None
+speed_cad_connections = []
+radio = None
+while True:
+    if heart:
+        if not hr_connection:
+            print("Running hr_connection")
```

Comment:
If you want to leave these print statements in, make them more understandable, like "Connecting to heart rate monitor", etc.